src/pages/[lang]/assistenza.tsx

```diff
@@ -1,226 +1,108 @@
-                color: "#17324D",
-              }}
-            >
-              06.4520.2323
-            </Link>{" "}
-            per assistenza dedicata dal lunedì al venerdì dalle 08:00 alle 20:00 e il
-            sabato dalle 08:00 alle 14:00.
-          </Typography>
-          <Button
-            variant="outlined"
-            color="primary"
-            component="a"
-            href="tel:0645202323"
-          >
-            Chiamaci
-          </Button>
-        </Stack>
-      </Grid>
-    </Grid>
-  );
-};
+export async function getStaticProps({
+  params,
+}: {
+  params: { lang: LangCode };
+}) {
+  const translations = getI18n(params.lang, ["common", "assistenza"]);
 
-const ContactInfoAssistenzaMittenti = () => {
-  return (
-    <Grid container spacing={0} sx={{ width: "100%" }}>
-      <Grid item xs={12} sx={{ backgroundColor: "#F5F5F5", padding: 8 }} display="grid" justifyContent="center" alignItems="center">
-        <Stack alignItems="center" justifyContent="center" spacing={2} maxWidth="500px">
-          <MailIcon sx={{ fontSize: "40px", color: "#0062C3" }} />
-          <Typography
-            variant="h6"
-            component="h2"
-            sx={{ color: "#17324D" }}
-            gutterBottom
-          >
-            Scrivici
-          </Typography>
-          <Typography
-            variant="body2"
-            sx={{ color: "#17324D" }}
-            textAlign="center"
-            marginBottom={2}
-          >
-            Richiedi assistenza come ente mittente via email scrivendo a{" "}
-            <Link
-              href={"mailto:" + PAGOPA_HELP_EMAIL}
-              sx={{
-                fontWeight: "bold",
-                textDecoration: "none",
-                color: "#17324D",
-              }}
-            >
-              {PAGOPA_HELP_EMAIL}
-            </Link>
-            .
-          </Typography>
-          <Button
-            variant="outlined"
-            color="primary"
-            component="a"
-            href={"mailto:" + PAGOPA_HELP_EMAIL}
-          >
-            Scrivici
-          </Button>
-        </Stack>
-      </Grid>
-    </Grid>
-  );
-};
+  return { props: { translations, lang: params.lang } };
+}
 
 const Assistenza: NextPage = () => {
+  const { t } = useTranslation(["common", "assistenza"]);
   const [currentTab, setCurrentTab] = useState({ index: 0, visible: true });
   const transitionDuration = 500;
   const containerRef = useRef(null);
-  const tabsData = getAssistenzaTabsData("tabs assistenza 1");
-  const headingTitleData = getAssistenzaHeadingTitleData(
-    "heading title assistenza 1"
-  );
+
+  const tabs = [
+    t("tab.1.title", { ns: "assistenza" }),
+    t("tab.2.title", { ns: "assistenza" }),
+  ];
+
+  const cardsContent: Array<AssistanceCardsProps> = [
+    {
+      // cards cittadini e imprese
+      cards: [
+        {
+          title: t("tab.1.card.1.title", { ns: "assistenza" }),
+          href: "/faq#send-come-accedere",

```

Comment:
these links are internal

Reply:
Eventually I fixed this issue also in this PR!